drivers/blktap2.py

```diff
@@ -1652,6 +1680,7 @@ def _activate_locked(self, sr_uuid, vdi_uuid, options):
 
         # Link result to backend/
         self.BackendLink.from_uuid(sr_uuid, vdi_uuid).mklink(dev_path)
+ 	self.linkNBD(sr_uuid, vdi_uuid)
```

Comment:
Looks to be some duplicated code here with what's on line ~ 1553. A cleanup in a subsequent commit would be good.